workers/unity/Assets/Fps/Scripts/GameLogic/Session/TrackPlayerSystem.cs

```diff
@@ -45,7 +46,10 @@ protected override void OnUpdate()
             }
             else
             {
-                SessionStatus = sessionGroup.GetComponentDataArray<Session.Component>()[0].Status;
+                Entities.With(sessionGroup).ForEach((ref Session.Component session) =>
```

Comment:
is there any chance that we have more than 1 entity matching this query?

Reply:
if you misconfigure it >< 
but you should always have at most one session entity in your deployment.

Reply:
@jessicafalk I think no?

Reply:
as mentioned: our snapshot always only contains at most one entity. A user can of course hack around with it and change it, but we assume you only have it at most once

Reply:
Is it oke by you to leave it like this?

Reply:
it's not nice, but ok to leave it like this for now. It's not a big issue